src/stan-users-guide/gaussian-processes.Rmd

```diff
@@ -1086,3 +1151,98 @@ generated quantities {
   Omega = L_Omega * L_Omega';
 }
 ```
+
+## Supported Covariance Functions
+
+Currently, Stan supports several build in covariance functions (kernels). These are the squared exponential, also known as the radial basis function or the gaussian kernel, a dot product or linear covariance function, the periodic covariance function, and three additional special cases of the Matern covariance function, that are the Matern 5/2 kernel, Matern 3/2 and the Matern 1/2 covariance function, or the exponential covariance function. We will review functional form, implementation and provide a visualization with simulated data below.
```

Comment:
"squared exponential, also known as the radial basis function or the gaussian kernel"
->
exponentiated quadratic
(the other names are mentioned later)